src/api/modelTypes.js

```diff
@@ -137,7 +143,12 @@ export type CrossRealmBot = {|
   is_bot: true,
   user_id: number,
 
-  // timezone included since commit 58ee3fa8c (in 1.9.0)
+  // The timezone field has been included at least since commit 58ee3fa8c (in
+  // 1.9.0). Tim mentions in 2020-02, at
```

Comment:
Not "at least" -- that's the actual commit that introduced it :slightly_smiling_face: